server/boot/06-create-season-and-week.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+var cron = require('node-cron');
+var moment = require('moment');
+var Promise = require('bluebird');
+var async = require('async');
+
+module.exports = function(app) {
+  console.log('Starting boot script 6');
+  var Nfl = app.models.Nfl;
+  var Group = app.models.Group;
+  var Season = app.models.Season;
+  var Week = app.models.Week;
+  var Score = app.models.Score;
+
+  // Setup scheduler for updating picks every hour
+  cron.schedule('0 0 * * * *', function() {
+    // Get current season and week in NFL
+
+    Nfl.findOne()
+    .then(function(nfl) {
+      console.log('found nfl', nfl);
+      Group.find().then(function(groups) {
+        console.log('available groups', groups);
+        async.eachLimit(groups, 1, function(group, cb) {
+          console.log('process group', group);
+          var newScores = [];
+          Season.upsertWithWhere({season: nfl.currentSeason, group: group.id},
+            {season: nfl.currentSeason, group: group.id})
+          .then(function(createdSeason) {
+            console.log('season upsert ', createdSeason);
+            group.currentSeason = createdSeason.id;
+            group.save();
+            // Create new week for season
+            Week.upsertWithWhere(
```

Comment:
Shouldn't you be returning this promise?

Reply:
No need because it being chained to the `.then`

Reply:
ok

Reply:
Needs to be rebased

Reply:
rebased